### ISSUE TITLE
Consumer product recall: Translation required on Category value

### ISSUE THREAD

**Issue by ndolter** [mmiron-HC/WST-RSAMS#127]
URL: https://hcdev.openplus.ca/fr/avis-rappel/test-alert-oct-13-new-recall
User Agent: Chrome
Language: French

Issue Description: There is a typo with the category 'Vehicles'.  According to Google Translate it should be 'Véhicule', but am not sure how reliable a source this is.

Steps to reproduce:
1. Using the url provided above, expand the 'Détail' section under 'Renseignements supplémentaires'
2. Visually scan
![image](https://user-images.githubusercontent.com/68117281/137542244-5244236b-512c-46c2-b360-04c97efdeb2c.png)

Expected results: All text on the page should match to that of the language of the page.

**Comment by echouin** [comment]
@ndolter fixed.

**Comment by hydegeoff** [comment]
confirmed issue is fixed.
